src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: 300ms;
```

Comment:
Please specify to which properties you want to apply the transition. 
For now, it applies to all the properties, and it's really bad for performance